screens/Login.tsx

```diff
@@ -0,0 +1,85 @@
+import { StyleSheet, TextInput, Text, View, Button } from 'react-native';
+import Form from 'react-native-form';
+import { useState } from 'react';
+
+export const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const submitForm = () => {
+    // query firebase
+  };
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.h1_heading}>
+        Welcome back!
+        <br />
```

Comment:
This is also breaking the mobile version -- we can't have HTML tags like <br> mixed into the React. You can achieve the same effect with this
```
{`Welcome back!\nPlease login.`}
```